app/uk/gov/hmrc/economiccrimelevyregistration/services/EmailService.scala

```diff
@@ -61,6 +62,9 @@ class EmailService @Inject() (emailConnector: EmailConnector)(implicit
       case (Some(firstContactName), Some(firstContactEmail), None, None)                                        =>
         sendEmail(firstContactName, firstContactEmail)
       case _                                                                                                    => throw new IllegalStateException("Invalid contact details")
+    }).recover { case e: Exception =>
+      logger.error(s"Failed to send email: ${e.getMessage}")
+      ()
```

Comment:
Should the exception be thrown once logged (otherwise aren't we just "swallowing" it here?)?

Reply:
I'll throw it for consistency with the reg backend for NRS failure but I don't think it'll make a difference materially as we log the exception above